test/fixtures/vite-lit/src/my-element.ts

```diff
@@ -0,0 +1,85 @@
+import { LitElement, css, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import './my-another-element'
+import './my-collision-element'
+
+/**
+ * An example element.
+ *
+ * @slot - This element has a slot
+ * @csspart button - The button
+ */
+@customElement('my-element')
+export class MyElement extends LitElement {
+  static styles = css`
+    :host {
+      display: block;
+      border: solid 1px gray;
+      padding: 16px;
+      max-width: 800px;
+    }
+    @unocss-include
```

Comment:
`@unocss-include` is a marker for the scanner to force include the file for scanning, while in this case, I think you are using it for another purpose. As it could confuse user, I'd suggest we rename it as `@unocss-placeholder`

Reply:
can you check this https://github.com/antfu/unocss/pull/281#discussion_r772723063? I include explitly `@unocss-import` on `IMPORT_CSS` then change the context filter to also check if including it. If we rename it we've braking changes...

Reply:
see this commit: https://github.com/antfu/unocss/pull/281/commits/490fe3f9984c79ed63c613dcd22ea98b2beb7e61